.dockerfiles/install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+install_deb() {
+	apt-get update
+	apt-get install -y --no-install-recommends \
+		curl=7.64.0-4 \
+		netcat=1.10-41.1 \
+		jq=1.5+dfsg-2+b1
+	apt-get install -y "/artifacts/netdata_${VERSION}_${ARCH}.deb"
+	apt-get clean
+	rm -rf /var/lib/apt/lists/*
```

Comment:
While this is generally good practice, I personally see little value in doing it. The images being used here are going to be deleted in very short order in a significant majority of cases, so the few dozen MB of extra space usage quite simply shouldn't matter enough to warrant spending time on reducing it.

Reply:
yes yes I agree; I'll remove these lines for now.

My thinking here long-term is we can in theory also replace the official Docker image that folks use to run NetData with _any_ of these. i.e: Our CI/CD pipelines can also publish these resulting images; we may not of course decide to publish all of them, just a few (like 1 alpine, 1 debian and 1 rhel) -- Let users decide which flavour of the Docker image they want to run up in their environments -- many of the upstream official images do this too :)